### PR TITLE
[FSSDK-9490] Prepare for release 5.0.0-beta.2

### PR DESCRIPTION
**Pull request by raju-opti**
## Summary
- release 5.0.0-beta2

## Issues
- FSSDK-9490

### CODE REVIEW

File: packages/optimizely-sdk/CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,15 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [5.0.0-beta.2] - July 19, 2023
```
Comment (mikechu-optimizely) [inline packages/optimizely-sdk/CHANGELOG.md:10]: Was there `-beta` changelog entries?

**Review by mikechu-optimizely — APPROVED** [review]
LGTM apart from the below conversation

File: packages/optimizely-sdk/CHANGELOG.md (hunk 1 above)
Comment (raju-opti) [reply packages/optimizely-sdk/CHANGELOG.md:10]: Yes. Just before the 4.9.4 entry

Comment (mikechu-optimizely) [reply packages/optimizely-sdk/CHANGELOG.md:10]: Oh. Ick...but okay.  Resolving conversation.

[the remark this answers is not in the document]
Comment (raju-opti) [reply packages/optimizely-sdk/CHANGELOG.md]: updated

File: packages/optimizely-sdk/lib/index.browser.tests.js [hunk 2]
```diff
@@ -188,7 +188,7 @@ describe('javascript-sdk (Browser)', function() {
         optlyInstance.onReady().catch(function() {});
 
         assert.instanceOf(optlyInstance, Optimizely);
-        assert.equal(optlyInstance.clientVersion, '5.0.0-beta');
+        assert.equal(optlyInstance.clientVersion, '5.0.0-beta.2');
```
[the remark this answers is not in the document]
Comment (raju-opti) [reply packages/optimizely-sdk/lib/index.browser.tests.js]: updated

**Review by jaeopt — APPROVED** [review]
LGTM

**Pull request merged**